rosbag2_cpp/include/rosbag2_cpp/writers/sequential_writer.hpp

```diff
@@ -117,6 +117,7 @@ class ROSBAG2_CPP_PUBLIC SequentialWriter
 
   /**
    * Write a message to a bagfile. The topic needs to have been created before writing is possible.
+   * Only writes message if timestamp is within start_time_ns and end_time_ns.
```

Comment:
@pfavr2 Could you please add a note that `start_time_ns` and `end_time_ns` are from `storage_options`?